csaf_2.1/prose/edit/src/schema-elements-02-props-02-document.md

```diff
@@ -627,8 +627,7 @@ Unique identifier for the document (`id`) of value type `string` with 1 or more
 ```
 
 Unique identifier for the document holds the Identifier.
-
-> It SHALL NOT start or end with a white space and SHALL NOT contain a line break.
+It SHALL NOT start or end with a white space and SHALL NOT contain a newline sequence.
```

Comment:
I would prefer space. The days of print are over and I assume we also do not want horizontal tabs and friends to be there either.

Reply:
I was using the term "white space" as this a fixed term in [EMCA 262](https://262.ecma-international.org/15.0/index.html#sec-white-space) which is the standard for [JSON patterns](https://json-schema.org/understanding-json-schema/reference/regular_expressions).
Should we add that to the terminology? (If so, how?)